src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/documentremoval/DocumentRemovalAboutToSubmitHandler.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler.documentremoval;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.finrem.caseorchestration.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.controllers.GenericAboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.finrem.caseorchestration.handler.FinremCallbackHandler;
+import uk.gov.hmcts.reform.finrem.caseorchestration.handler.FinremCallbackRequest;
+import uk.gov.hmcts.reform.finrem.caseorchestration.mapper.FinremCaseDetailsMapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseDetails;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.documentremoval.DocumentRemovalService;
+
+
+@Slf4j
+@Service
+public class DocumentRemovalAboutToSubmitHandler extends FinremCallbackHandler {
+
+    private final DocumentRemovalService documentRemovalService;
+
+    @Autowired
```

Comment:
@Autowired is no longer required where there is only one constructor.
https://docs.spring.io/spring-framework/reference/core/beans/annotation-config/autowired.html